pkg/sql/colexec/connector/connector.go

```diff
@@ -43,11 +43,11 @@ func Call(_ int, proc *process.Process, arg any, _ bool, _ bool) (bool, error) {
 	select {
 	case <-proc.Ctx.Done():
 		proc.PutBatch(bat)
-		logutil.Infof("proc context done during connector send")
+		logutil.Debugf("proc context done during connector send")
```

Comment:
This error is probably a real error, like a network disconnection or something. I think it needs to be info

Reply:
reverted